### PR TITLE
Kernel: Add _SC_CLK_TCK to sysconf.

### PR DESCRIPTION
**Pull request by thatdutchguy**
Makes patching unnecessary for the Hatari port (see https://github.com/SerenityOS/serenity/pull/5812)

### CODE REVIEW

**Comment by thatdutchguy** [comment]
Updated to fix linting issues

File: Kernel/Syscalls/sysconf.cpp [hunk 1]
```diff
@@ -44,6 +44,9 @@ KResultOr<long> Process::sys$sysconf(int name)
         return TTY_NAME_MAX;
     case _SC_GETPW_R_SIZE_MAX:
         return 4096; // idk
+    case _SC_CLK_TCK:
+        // FIXME: should return Number of clock intervals per second for times()
+        return 100; // seems to be 100 on x86_64
```
Comment (RealKC) [inline Kernel/Syscalls/sysconf.cpp:49]: Not really a kernel programmer, but wouldn't calling https://github.com/SerenityOS/serenity/blob/master/Kernel/Time/TimeManagement.h#L66 be more appropriate?

Comment (thatdutchguy) [reply Kernel/Syscalls/sysconf.cpp:49]: Thanks, that does seem more appropriate, let me give it a go.

**Comment by thatdutchguy** [comment]
That seems to work, and now it returns 250 👍

**Pull request merged**